dot/network/message.go

```diff
@@ -354,3 +360,79 @@ func (cm *ConsensusMessage) Hash() (common.Hash, error) {
 	}
 	return common.Blake2bHash(encMsg)
 }
+
+// NewSingleBlockRequestMessage returns a request to retrieve a single block
+func NewSingleBlockRequestMessage(blockHash common.Hash, requestedData byte) *BlockRequestMessage {
+	one := uint32(1)
+	return &BlockRequestMessage{
+		RequestedData: requestedData,
+		StartingBlock: *variadic.MustNewUint32OrHash(blockHash),
+		Direction:     Descending,
+		Max:           &one,
```

Comment:
Why are we using `*uint32` instead using `uint32`? is there any benefit?

Reply:
I was trying to understand if this pointer was needed for protobuf encoding/decoding but no, it can be just a normal `uint32`, I can address this in another PR as it seems out of scope for this PR, sounds good?

Reply:
Ok np let's add a TODO to remember it